src/replica/ConfigurationSchema.h

```diff
@@ -131,6 +135,42 @@ class ConfigurationSchema {
     }
 
 private:
+    /**
+     * @brief Retreive a value of the parameter's attribute (allow default value).
+     * 
+     * @tparam T The type of the attribute's value.
+     * @param category The name of the parameter's category.
+     * @param param The name of the parameter within its category.
+     * @param attr The name of the attribute.
+     * @param defaultValue The default value to be returned if no such parameter
+     *   or its attribute was fount.
+     * @return T The value of the attribute (or the default value).
+     */
+    template <typename T>
+    static T _attributeValue(std::string const& category, std::string const& param,
+                             std::string const& attr, T const& defaultValue) {
+        if (_schemaJson.count(category)) {
```

Comment:
Maybe `contains()` instead of `count()`? Though in this case `find()` should be better.

Reply:
Yes, that one would work: https://json.nlohmann.me/api/basic_json/contains/